eventbus/src/main/java/pl/edu/uj/kimage/eventbus/VertXEventBus.java

```diff
@@ -0,0 +1,36 @@
+package pl.edu.uj.kimage.eventbus;
+
+public class VertXEventBus implements EventBus {
```

Comment:
This is a technology specific implementation. Why it shouldn't have specific technology in the name?